### DISCUSSION TITLE
Node failure on supermuc

### DISCUSSION THREAD

**Discussion by NicoSchlw** [SeisSol/SeisSol#465] [Q&A] [answered]
Hi, my seissol job on supermuc failed two times due to node failure after running already for quite some time. Does this mean there was a hardware issue or could the failure be related to my setup?

**Comment by krenzland** [comment, marked as answer]
Could be either option. Which equation/number of elements/ how many nodes are you using?
Does it happen every time?
Sometimes node failures occur when you allocate too much memory.
You can also try and add the following line to your batch job script:
> #SBATCH --ear=off

**Comment by NicoSchlw** [comment]
Thanks for the fast reply, it's DR and the mesh has 27mio elements. I use 400nodes, so I guess memory shouldn't be a problem? I already ran similar models with the same mesh successfully, but this exact setup failed 2/2. Those are the log/parameter/submission files:

[1927009.cas.err.txt](https://github.com/SeisSol/SeisSol/files/7926781/1927009.cas.err.txt)
[1927009.cas.out.txt](https://github.com/SeisSol/SeisSol/files/7926782/1927009.cas.out.txt)
[parameters_200s.txt](https://github.com/SeisSol/SeisSol/files/7926784/parameters_200s.txt)
[submission_100s.txt](https://github.com/SeisSol/SeisSol/files/7926785/submission_100s.txt)

**Reply by krenzland** [reply]
The setup looks reasonable. Only difference that I can detect is that on SuperMUC-NG mpiexec is preferred over srun (see [here](https://doku.lrz.de/display/PUBLIC/Job+Processing+with+SLURM+on+SuperMUC-NG#JobProcessingwithSLURMonSuperMUCNG-BatchJobExamples)).
You should also try with ear=off (this is sometimes buggy on NG).
By the way, you should probably use slightly fewer nodes, it's not scaling *that* well (maybe ~250?). But the performance looks _fine_ just not _great_.

If it happens again, I'd probably submit a ticket to LRZ. Pretty sure that that problem is not caused by SeisSol.

**Reply by NicoSchlw** [reply]
Ok, I will try this, thanks a lot!
